### PR TITLE
Adds chameleon belt to cham kit

### PR DESCRIPTION
**Pull request by improvedname**
So that the chamkit can be finally whole.

## Changelog
:cl:
tweak: Chamkits now include an chameleon belt
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -460,6 +460,7 @@
 	new /obj/item/clothing/mask/chameleon(src)
 	new /obj/item/clothing/neck/chameleon(src)
 	new /obj/item/storage/backpack/chameleon(src)
+	new /obj/item/storage/belt/chameleon
```
Comment (Fikou) [inline code/game/objects/items/storage/uplink_kits.dm]: you forgot (src)

**Comment by itseasytosee** [comment]
I was going to make a chameleon holster that came with the syndie revolver.

**Comment by Fikou** [comment]
> I was going to make a chameleon holster that came with the syndie revolver.

you can still do that

**Comment by HomocidlePancakes** [comment]
Bring back the chamelion gun!

**Comment by improvedname** [comment]
@HomocidlePancakes Chameleon gun was removed due to performace issues

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**